app/imports/components/apply/form-test.jsx

```diff
@@ -9,9 +9,25 @@ import { FormSchema } from '/imports/schemas/TestForm.js';
 
 // Import methods
 import { saveApplicationSection } from '/imports/api/methods.js';
+import { saveApplyPosition } from '/imports/api/methods.js';
```

Comment:
Gotta fix this to
`import { saveApplicationSection, saveApplyPosition } from '/imports/api/methods.js';`